### PR TITLE
Uniform KB and Gallery

### LINKED ISSUE #823: Uniformization of Knowledge Base/Gallery
**Description**
Try to uniform KB and Gallery.

**Expected change**

- Uniform images with the same style (center, correct size, borders)
- Uniform button to download files/zip
- Uniform structure inside each section:
   - *src* folder to put code and files
   - *images* folder to put images, gif and videos
- Uniform titles

### PR DESCRIPTION
**Pull request by FlorianJacta**
Resolves: #823 

- Uniform images with the same style (center, correct size, borders)

image

- Uniform button to download files/zip

image

- Uniform structure inside each section:
   - *src* folder to put code and files
   - *images* folder to put images, gif and videos

image

- Uniform titles

### CODE REVIEW

**Review by jrobinAV — COMMENTED** [review]
Please factorize the change in the style sheet.

**Comment by FlorianJacta** [comment]
@jrobinAV Done!

**Review by jrobinAV — APPROVED** [review]

File: docs/knowledge_base/tutorials/2_scenario_management_overview/index.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ def predict(historical_temperature: pd.DataFrame, date_to_forecast: str) -> floa
 ```
 The scenario can be represented as the following graph:
 
-![Simple scenario](images/config.svg){ width=80% style="margin:auto;display:block;border: 4px solid rgb (210,210,210);border-radius:7px" }
+![Simple scenario](images/config.svg){ width=80% : .tp-image }
```
Comment (FabienLelaquais) [inline docs/knowledge_base/tutorials/2_scenario_management_overview/index.md:74]: Why is this 'image' and not 'image-border'?

**Review by FabienLelaquais — APPROVED** [review]

File: docs/knowledge_base/tutorials/2_scenario_management_overview/index.md (hunk 1 above)
Comment (FlorianJacta) [reply docs/knowledge_base/tutorials/2_scenario_management_overview/index.md:74]: I didn't put a border to *svg* files that represent pipelines. Having a border would seem strange in my opinion 
image

**Pull request merged**